### ISSUE TITLE
We are getting "I need 'Embed Links' and 'Add Reactions' permissions!" despite those being set

### ISSUE THREAD

**Issue by joelmichael** [BibleBot/BibleBot#57]
It happens consistently and repeatedly in our Discord, starting tonight, apparently.

BibleBot#1110BOT
PLAYING A GAME
BibleBot v7.0.0 | Shard: 2 / 2

**Comment by vypr** [comment]
Yes, the bot requires new permissions of 'Read Message History' and 'Manage Messages' in order to use reactions completely, but I missed one of the error messages and so it only shows this.

Either update the bot's role to have these permissions, or kick it and reinvite it with the new link that has the permissions automatically set up: https://discordapp.com/oauth2/authorize?client_id=361033318273384449&scope=bot&permissions=93248

**Comment by joelmichael** [comment]
I don’t run the discord but the admin is reporting it’s not working even after setting the additional perms

**Comment by joelmichael** [comment]
We are also unsure why reactions are necessary, pls update readmd

**Comment by vypr** [comment]
Discord *just* had some downtime so the bot was up and down for a bit, it should be working fine now.

Reactions are used as part of +search and +versions as they are paged embeds (test it out in a different server to see what I mean).

**Comment by vypr** [comment]
Worst-case scenario, I don't mind joining your server and getting it fixed.

I'll update the docs with a permission section soon.

**Comment by joelmichael** [comment]
It's fixed now for some reason 🤷‍♂️ Thanks
